meta-resin-common/recipes-core/ucl/ucl_1.03.bb

```diff
@@ -0,0 +1,21 @@
+HOMEPAGE = "http://www.oberhumer.com/opensource/ucl/"
+SUMMARY = "Data compression library"
+
+SRC_URI = "http://www.oberhumer.com/opensource/ucl/download/ucl-1.03.tar.gz"
+
+SRC_URI[md5sum] = "852bd691d8abc75b52053465846fba34"
+SRC_URI[sha256sum] = "b865299ffd45d73412293369c9754b07637680e5c826915f097577cd27350348"
+
+inherit native
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=dfeaf3dc4beef4f5a7bdbc35b197f39e"
+
+do_configure_append() {
+    ${S}/configure --prefix=${D}${prefix}
+}
+
+do_install() {
+   oe_runmake install
+   find "${D}" -ignore_readdir_race -name "*.la" -delete
```

Comment:
i am sort of suer. Because I cannot find any code in the tasks that native defines to strip the TMPDIR path from those files, but the QA check is there.